lavamoat/browserify/mmi/policy.json

```diff
@@ -34,11 +34,35 @@
         "@ensdomains/content-hash>cids>multibase>@multiformats/base-x": true
       }
     },
+    "@ensdomains/content-hash>cids>multicodec": {
```

Comment:
Not sure why some of these lavamoat policies changed.  `chart.js` and `react-chartjs-2` are the only new dependencies.  And I'm just using `@metamaskbot update-policies` on the PR.